__tests__/app.test.js

```diff
@@ -118,4 +118,54 @@ describe('3. Get api/articles/:article_id', () => {
                 })
         })
     })
+})
+
+describe('4. GET /api/articles/:article_id/comments', () => {
+    describe('a. status 200 & data', () => {
+        it('returns 200 status code & an array back to the user where the data starts with the latest comment', () => {
+            return request(app)
+            .get('/api/articles/3/comments')
+            .expect(200)
+            .then((res) => {
+                const articles = res.body;
+                expect(articles).toBeInstanceOf(Array);
+                expect(articles.length).toBe(2);
+                articles.forEach(article => {
+                    expect(article).toEqual(
+                        expect.objectContaining({
+                            article_id : 3,
+                            author : expect.any(String),
+                            body : expect.any(String),
+                            created_at : expect.any(String),
+                            votes : expect.any(Number)
+                        })
+                    )
+                })
+                expect(articles).toBeSorted({ descending: true })
+            })
+        })
+
+        it('returns 200 status code & empty array when a valid ID is passed but contains no comments', () => {
+            return request(app)
+                   .get('/api/articles/7/comments')    
+                   .expect(200)
+                   .then( (res) => {
+                        const articles = res.body;
+                        expect(articles).toEqual([])
+                   })
+        })
+    })
+
+    describe('b. error handling', () => {
+
+        it('send a 400 status code when user inputs the wrong datatype for the ID', () => {
+            return request(app)
+                .get('/api/articles/article3/comments')
+                .expect(400)
+                .then ((res) => {
+                    const {msg} = res.body;
+                    expect(msg).toBe('Bad Request')
+                })
+        })
+    })

```

Comment:
just missing one more test for the `404` when a user inputs an `article_id` which is not found in the database, for example `999` or `0`